moto/applicationautoscaling/models.py

```diff
@@ -135,7 +135,17 @@ def _target_params_are_valid(namespace, r_id, dimension):
         try:
             valid_dimensions = [d.value for d in ScalableDimensionValueSet]
             d_namespace, d_resource_type, scaling_property = dimension.split(":")
-            resource_type, cluster, service = r_id.split("/")
+            if r_id.startswith("arn:aws:comprehend"):
```

Comment:
Can we extract this into a separate method, and add some comments to make it clear what r_id should look like?

Reply:
Done.  I think this build is OK - it's the recurring batch issue.  It built fine on my Travis.